initial_positions:
  joint_1: 0.0
  joint_2: 1.98
  joint_3: -2.27
  joint_4: 0.0